docs/JK.Common/Extensions/DateNameExtensions.md

```
@@ -0,0 +1,115 @@
+**Returns:** Full day name.
+
+## MonthName *(Inherited)*
+
+**Signature:** `MonthName`
+
+**Summary:**
+Gets the full name of a month.
+
+**Returns:** Full month name.
+
+## AbbreviatedDayName *(Inherited)*
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)